src/components/ActionPanel.tsx

```diff
@@ -73,24 +85,35 @@ export default class ActionPanel extends React.Component<Props> {
                         cost="-5 STA"
                         onClick={this.wander}
                         paused={paused}
+                        locked={stamina <= 0}
                         cooldown={1000}
                     />
                     {learned.size ?
                      <ActionButton
                          label="Meditate"
-                         cost="-5 STA"
+                         benefit="+5 STA"
                          onClick={this.meditate}
                          paused={paused}
-                         cooldown={2000}
+                         cooldown={1000}
                      />
                      : false}
-                    <ActionButton label="Transliterate" benefit="+¥" paused={paused} />
+                    {learned.size ?
```

Comment:
I would rather this not appear until the quest is finished.

Reply:
@tyleretzel

Reply:
actually i changed it: 2a2c23b